hw/ip/otbn/data/otbn.hjson

```diff
@@ -364,7 +337,7 @@
     } // register : start_addr
     { name: "FATAL_ALERT_CAUSE",
```

Comment:
Once we have decided on a naming for the errors, I'd align the names in FATAL_ALERT_CAUSE with the ones in ERR_BITS.